_component_design/checkboxes.md

```diff
@@ -3,10 +3,9 @@ title: Checkboxes
 parent: Inputs and Controls
 layout: component
 category: Components
-usage: |
-  Helix controls allow users to select options in a variety of ways including
-  radio buttons, checkboxes, switches and more.
-preview-image: components/preview-selectors.svg
+usage: >
+  Checkboxes are used to change settings or bulk item selection. Checkboxes allow the user to select zero, one or several items.
```

Comment:
Nit: Comma missing:
```
Checkboxes allow the user to select zero, one, or several items.
```

Reply:
Fixed in commit fe6a7cf and added to this PR. Additionally, I know Alex has an open branch on this page so i'm sure her word choices will be better than mine. We can merge her PR in after this one is merged.